ios/Plugin.xcodeproj/project.pbxproj

```diff
@@ -287,11 +287,13 @@
 				"${PODS_ROOT}/Target Support Files/Pods-PluginTests/Pods-PluginTests-frameworks.sh",
 				"${BUILT_PRODUCTS_DIR}/Capacitor/Capacitor.framework",
 				"${BUILT_PRODUCTS_DIR}/CapacitorCordova/Cordova.framework",
+				"${BUILT_PRODUCTS_DIR}/RadarSDK/RadarSDK.framework",
```

Comment:
I'm not sure why this and line 296 weren't there before, but they were automatically added.